Bottom: 5bd82bf2ac6b5b0a857dd444ab29e3d9a4da8802
Top:    952a2b5311e2dd336ca62369252066de622b0b70
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 20:10:26 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
